core/server/data/migrations/versions/4.17/02-add-offers-table.js

```diff
@@ -0,0 +1,18 @@
+const utils = require('../../utils');
+
+module.exports = utils.addTable('offers', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    name: {type: 'string', maxlength: 191, nullable: false},
+    code: {type: 'string', maxlength: 191, nullable: false, unique: true},
+    product_id: {type: 'string', maxlength: 24, nullable: false},
+    interval: {type: 'string', maxlength: 50, nullable: false, validations: {isIn: [['monthly', 'yearly']]}},
```

Comment:
These intervals are incorrect btw - it should be `'month'` and `'year'`

Reply:
👍 Updated